src/mixins/filing-template-mixin.ts

```diff
@@ -344,7 +345,7 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
         business: {
           identifier: this.getTempId,
           naics: this.getRegistration.naics,
-          taxId: this.getRegistration.businessNumber
+          taxId: this.getRegistration.businessNumber || undefined // can't be null
```

Comment:
Using `undefined` makes this prop NOT part of the resulting JSON.

(This magic is done by JS' internal object <-> JSON conversion.)